tests/languages/php/attribute_feature.test

```diff
@@ -0,0 +1,364 @@
+// #[Foo]
+
+#[]
+
+#[
+	// something ]
+	Foo,
+	/* something
+	else #[] */
+	Bar
+	# shell comments aren't confusing at all in here
+]
+
+#[Foo([0, 1])]
+
+#[
+	Foo(
+		[
+			1 => [0, 1],
+			2 => [2, 3],
+			3 => [
+				[0, 1],
+				[2, 3]
+			]
+		]
+	)
+]
+
+#[Foo]
+#[Foo\Bar\Baz]
+#[Route(Http::POST, '/products/create', 1)]
+#[
+	Http\Route(Http::POST, '/products/create', 1),
+	Foo\Bar\Baz,
+	AttributeFoo('value')
+]
+
+#[A1(1), A1(2), A2(3)]
+class Foo {
+	public function foo(#[A1(5)] $a, #[A1(6)] $b) { }
+}
+
+$object = new #[A1(7)] class () {};
+
+function foo(
+    #[Attribute] $param1,
+    $param2
+) {}
+
+$f1 = #[ExampleAttribute] function () {};
+
+$ref = new \ReflectionFunction(#[A1] #[A2] function () { });
+
+#[DeprecationReason('reason: <https://some-website/reason?>')]

```

Comment:
Without an opening `<?php` tag, this test isn't very interesting. The intention behind this was to trip `phpPattern`. Please make a new test file with the following code:

```php
<?php
#[DeprecationReason('reason: <https://some-website/reason?>')]
function main() {}
?>
```

Reply:
Test case added.